<?xml version="1.0"?>
<launch>
	<!-- start environment modelling -->
	<node pkg="cob_env_model" type="extract_features_node" name="extract_features" ns="cob_env_model" cwd="node" respawn="false" output="screen" launch-prefix="/usr/bin/gdb --args ">
		<remap from="point_cloud2" to="/sensor_fusion/shared/colored_point_cloud"/>
	</node>
</launch>

